### PR TITLE
Updated fonts sizes in design guidance

### PR DESCRIPTION
**Pull request by dmcalester**
Font sizes were in design guidelines were not updated to reflect Astro 5/KM1.2 changes

### CODE REVIEW

File: _content/design-guidelines/typography.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ Astro supports three levels of headline text (h1, h2, h3). CSS properties are au
 
 | Heading | Weight | Markup | REM   | PX  | Class |
 | ------ | ------ | ------ | ----- | --- | ----- |
-| Heading Level 1     | Light  | `<h1>` | 2.375 | 38  | `.h1` |
-| Heading Level 2     | Light  | `<h2>` | 1.75  | 28  | `.h2` |
-| Heading Level 3     | Light  | `<h3>` | 1.375 | 22  | `.h3` |
+| Heading Level 1     | Normal  | `<h1>` | 2.125 | 34  | `.h1` |
+| Heading Level 2     | Normal  | `<h2>` | 1.5  | 24  | `.h2` |
+| Heading Level 3     | Medium  | `<h3>` | 1.25 | 20  | `.h3` |
```
Comment (Full-lifey) [inline _content/design-guidelines/typography.md:21]: Can we have something more specific in weights? Normal is a CSS value, but medium is not. Can we change these to specific values that developers can use?

File: _content/design-guidelines/typography.md [hunk 2]
```diff
@@ -30,11 +30,11 @@ The default font size in Astro is 1rem/16px. Astro supports four additional font
...
-| Extra large | Normal | 1.125 | 18  | `.xl` |
-| Normal/large | Normal | 1     | 16  | --    |
-| Medium | Normal | 0.875 | 14  | `.md` |
-| Small | Normal | 0.75  | 12  | `.sm` |
-| Extra small | Normal | 0.65  | 10  | `.xs` |
+| Extra large | Normal (400) | 1.125 | 18  | `.xl` |
+| Normal/large | Normal (400) | 1     | 16  | --    |
+| Medium | Normal (400) | 0.875 | 14  | `.md` |
+| Small | Normal (400) | 0.75  | 12  | `.sm` |
+| Extra small (400) | Normal | 0.65  | 10  | `.xs` |
```
Comment (Full-lifey) [inline _content/design-guidelines/typography.md:37]: This 400 needs to shift into the next column.

**Review by Full-lifey — CHANGES_REQUESTED** [review]
Thanks for the edits. Just need to move the number on line 37

**Review by Full-lifey — APPROVED** [review]

**Comment by amalyah** [comment]
@dmcalester Is this PR attached to a ticket?

**Comment by amalyah** [comment]
Not directly related to this PR's changes, but is the size of the code text intentional? It's a lot bigger compared to our normal text size on Astro UXDS.

**Comment by amalyah** [comment]
Do we want to include letter spacing in this table? In the design files we have specific letter spacing for each one (as mandated by KM in many instances).

File: _content/design-guidelines/typography.md [hunk 3]
```diff
@@ -30,11 +30,11 @@ The default font size in Astro is 1rem/16px. Astro supports four additional font
...
-| Extra large | Normal | 1.125 | 18  | `.xl` |
-| Normal/large | Normal | 1     | 16  | --    |
-| Medium | Normal | 0.875 | 14  | `.md` |
-| Small | Normal | 0.75  | 12  | `.sm` |
-| Extra small | Normal | 0.65  | 10  | `.xs` |
+| Extra large | Normal (400) | 1.125 | 18  | `.xl` |
+| Normal/large | Normal (400) | 1     | 16  | --    |
+| Medium | Normal (400) | 0.875 | 14  | `.md` |
+| Small | Normal (400) | 0.75  | 12  | `.sm` |
+| Extra small | Normal (400) | 0.65  | 10  | `.xs` |
```
Comment (amalyah) [inline _content/design-guidelines/typography.md:37]: What do we use this font size for? I don't have it in the design files which means we probably have an inconsistency.

Comment (dmcalester) [reply _content/design-guidelines/typography.md:37]: It’s likely not used, just created for utility.

**Comment by dmcalester** [comment]
> Do we want to include letter spacing in this table? In the design files we have specific letter spacing for each one (as mandated by KM in many instances).

Will create a ticket, this was more of an immediate need to fix. I think that kind of guidance is worth a go-back, it’s incredibly specific and ties the hands of designers.

**Pull request merged**